.env.example

```diff
@@ -0,0 +1,8 @@
+VITE_WALLECT_CONNECT_PROJECT_ID=xxxxx
+VITE_NETWORK_ID=80001
+VITE_INFURA_ID=xxxxx
+VITE_INFURA_SECRET=xxxxxx
+VITE_PLATFORM_ID=1
```

Comment:
need to clean what you don't need here: 
- VITE_PLATFORM_ID, VITE_POH_SUBGRAPH_URL, VITE_LENS_URL